parlai/crowdsourcing/tasks/turn_annotations_static/conf/example.yaml

```diff
@@ -1,5 +1,9 @@
 #@package _global_
 mephisto:
+  architect:
```

Comment:
Hmm - it'd be good to keep these lines in the `run.py` file for now, for consistency with the other Mephisto tasks. I've made a note to hopefully move these lines into the `example.yaml` files of *all* tasks once we move to Hydra 1.1